pom.xml

```diff
@@ -510,6 +510,8 @@
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
                     <scmVersionType>tag</scmVersionType>
                     <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
+                    <scmVersion>CHK-2459-remove-postepay-gateway</scmVersion>
```

Comment:
Remember to restore this

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/425/commits/349bd8ee9aef6b2441b5caa9367e35ffe3807b4b